frontend/routes/manage/Questionnaire.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div>
+        <div class="space-between">
+            <h1>Responses</h1>
+            <div>
+                <router-link to="./message"><Button content="Message"></Button></router-link>
+                <TextInput placeholder="Search bar..."></TextInput>
+            </div>
+        </div>
+        <Table :rows="tableRows" :shownColumns="tableColumns"></Table>
+        <div class="space-between">
+            <Dropdown name="Show n rows">
+                <option selected>5</option>
+                <option>20</option>
+            </Dropdown>
+        </div>
+        <br>
+        <div class="space-between">
+            <h1>Questions</h1>
+            <h1>(un)lock</h1>
```

Comment:
Can this (and similar instances) be bumped down to an `<h2>`? I think the Figma has "Questions" at 3rem and "(un)lock" at 2.5rem